modulemd/tests/ModulemdTests/translation_helpers.py

```diff
@@ -137,6 +138,36 @@ def test_translations_from_catalog(self):
                                 profile.get_description(
                                     str(catalog.locale)), msg_string)
 
+    @mock.patch('ModulemdTranslationHelpers.Utils.KojiSessionMock')
+    def test_index_from_tags(self, mock_session):
+        tags = ['f29']
+
+        # Create a Modulemd.ModuleIndex object containing the YAML stream
+        idx = Modulemd.ModuleIndex.new()
+        self.assertIsNotNone(idx)
+        ret, failures = idx.update_from_file(
+            "%s/f29.yaml" %
+            (os.getenv('TEST_DATA_PATH')), True)
+        self.assertTrue(ret)
+        self.assertTrue(len(failures) == 0)
+
+         # Test when koji returns proper modules
+        index = Utils.get_index_from_tags(mock_session, tags)
+        self.assertIsNotNone(index)
+        mock_session.listTagged.assert_called_once_with('f29')
+        self.assertEqual(idx.dump_to_string(), index.dump_to_string())
+        mock_session.reset_mock()
+
+        # Test when koji returns no modules
+        mock_session.listTagged.return_value = []
+        mock_session.getBuild.return_value = None
+
+        index = Utils.get_index_from_tags(mock_session, tags)
+        self.assertIsNotNone(index)
+        mock_session.listTagged.assert_called_once_with('f29')
+        module_names = index.get_module_names()
+        self.assertFalse(module_names)
```

Comment:
What is this supposed to mean?

Reply:
This test says that a `ModulemdIndex` object will me created but it would be empty.

Reply:
I added a test for a case when `listTagged` wouldn't return anything. In that case, there would be an empty `ModulemdIndex` object.